### PR TITLE
Throw exception when creating Filament user without panel

### PR DESCRIPTION
**Pull request by stfnxd**
- [ Yes ] Changes have been thoroughly tested to not break existing functionality.
- [ No ] New functionality has been documented or existing documentation has been updated to reflect changes.
- [ No ] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

File: packages/panels/src/Commands/MakeUserCommand.php [hunk 1]
```diff
@@ -87,11 +87,17 @@ protected function getUserModel(): string
 
     public function handle(): int
     {
+        
         $this->options = $this->options();
 
-        $user = $this->createUser();
+        try{
```
Comment (zepfietje) [inline packages/panels/src/Commands/MakeUserCommand.php]:
```suggestion
```

File: packages/panels/src/Commands/MakeUserCommand.php [hunk 2]
```diff
@@ -87,11 +87,17 @@ protected function getUserModel(): string
 
     public function handle(): int
     {
+
```
Comment (zepfietje) [inline packages/panels/src/Commands/MakeUserCommand.php]:
```suggestion
```

File: packages/panels/src/Commands/MakeUserCommand.php [hunk 3]
```diff
@@ -89,9 +89,13 @@ public function handle(): int
     {
         $this->options = $this->options();
 
-        $user = $this->createUser();
-
-        $this->sendSuccessMessage($user);
+        try{
+            $user = $this->createUser();
+            $this->sendSuccessMessage($user);
+
```
Comment (zepfietje) [inline packages/panels/src/Commands/MakeUserCommand.php:91]:
```suggestion
```

**Comment by danharrin** [comment]
Instead of catching all exceptions, why not do a pre-emptive check before and see if the panel exists? Catching all exceptions does not sound like it will give an accurate solution in many cases.

**Comment by stfnxd** [comment]
Code should now be optimal, and pre-emptive check if the panel exist.

Unsure if you prefere ! or === null, so made ===null.

Sry for late response

**Review by zepfietje — CHANGES_REQUESTED** [review]
Please undo all formatting changes and empty lines that were added/removed.

**Comment by zepfietje** [comment]
There's still a couple more, @stfnxd.
Please check the diff.

**Comment by stfnxd** [comment]
Is this what you mean? @zepfietje 

If not, i am unsure what you are looking for, any chance you can link me some documentation on the formatting?

**Comment by danharrin** [comment]
Thanks

**Pull request merged**